### PR TITLE
Log fixes

### PR DESCRIPTION
**Pull request by jgoizueta**
See #9624 #9623

### CODE REVIEW

**Comment by jgoizueta** [comment]
@iriberri please review

**Comment by iriberri** [comment]
LGTM, let's see if @ethervoid can add something else! :)

File: spec/models/log_spec.rb [hunk 1]
```diff
@@ -15,8 +15,8 @@
       timestamp = Time.now.utc
 
       log = Log.new({ type: type })
-      log.append(text1, timestamp)
```
Comment (ethervoid) [inline spec/models/log_spec.rb]: What about to remove the timestamp because is the same that the default value in the method definition

**Comment by ethervoid** [comment]
Looks good to me. Good catch!

**Pull request merged**